src/report_modules/CMakeLists.txt

```diff
@@ -7,3 +7,4 @@
 
 add_subdirectory(report_module_text)
 add_subdirectory(report_module_gui)
+add_subdirectory(report_module_github_ci)
```

Comment:
What if we add an option to enable the module ([reference](https://cmake.org/cmake/help/latest/command/option.html))?

Maybe this will help us to reach a merging point faster.

Reply:
In my opinion this does not really change anything. Also an option needs to be maintained and working. I am not really a big fan of cmake options, because I think it makes everything harder to understand and use.

But I am not even a part of this project. I just wanted to contribute the GitHub pipeline output, because we have an application for this in [OpenMSL](https://github.com/openmsl). So I'll let you work out the details ;)

Reply:
I can understand. As long as it has no additional dependecies, we can consider it for _always compile it_.  But I'll refer the discussion to the Framework workgroup to decide.

Reply:
CCB: Let us compile it always.